pool/position.gno

```diff
@@ -11,12 +11,30 @@ import (
 	u256 "gno.land/p/gnoswap/uint256"
 )
 
+var (
+	Q128 = u256.MustFromDecimal(consts.Q128)
+)
+
 // positionGetKey generates a unique key for a position based on the owner's address and the tick range.
 func positionGetKey(
 	owner std.Address,
 	tickLower int32,
 	tickUpper int32,
 ) string {
+	if !owner.IsValid() {
+		panic(addDetailToError(
+			errInvalidAddress,
+			ufmt.Sprintf("position.gno__positionGetKey() || invalid owner address %s", owner.String()),
+		))
+	}
+
+	if tickLower > tickUpper {
+		panic(addDetailToError(
+			errInvalidTickRange,
+			ufmt.Sprintf("position.gno__positionGetKey() || tickLower(%d) is greater than tickUpper(%d)", tickLower, tickUpper),
```

Comment:
This is same with above comment
Error code is `errInvalidInput`. Details are included in error message.
So we know what situation is happend

Reply:
Based on the policy for error wording (using detailed error codes), this code does not need to be modified.